packages/react/src/components/Masthead/MastheadProfile.js

```diff
@@ -47,6 +48,16 @@ const MastheadProfile = ({
     <HeaderGlobalAction
       aria-label="User Profile"
       data-autoid={`${prefix}--masthead__profile`}
+      onMouseEnter={() =>
```

Comment:
We really want to fire an event like this every time we hover? If that's true should we consider adding a delay before triggering it again to avoid a number of hits and impact on performance? I guess I assumed we would want to track clicks.

Reply:
Looking at the documentation a little closer I think they it says hover because the old masthead worked using hover. I would imagine keeping with the same heart/vein because the dropdown interacts via click we would want to use clicks here instead.

cc: @jeffchew @kennylam

Reply:
yeah it's a bit much to do on hover - a delay might be a good idea.

I talked to @kennylam about it yesterday the dropdown. Currently the carbon masthead dropdowns only appears on click but we're going to create another component or wrapper so that it appears on hover

Reply:
Taking that information into account I can see an argument for keeping hover around until that comes but seeing that we'll probably release 1.0.0 before that happens I think maybe we should switch it to what's actually happening, and then we could change it when we get the hover states in? Thoughts?

Reply:
I'm fine with switching to click for now, we just need to let Juan know.
cc: @kennylam @ljcarot @photodow @annawen1

Reply:
ok, I'll have to add some event handlers for keyboard events as well